### DISCUSSION TITLE
2005/12/30/%E4%BC%91%E5%81%87%E4%B8%A4%E5%91%A8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#71] [Announcements]
# 2005/12/30/%E4%BC%91%E5%81%87%E4%B8%A4%E5%91%A8

好长时间没更新我的 space 了。这周和下周我都在黑龙江老家休假。

https://blog.qizhen.xyz/2005/12/30/%E4%BC%91%E5%81%87%E4%B8%A4%E5%91%A8

**Comment by ruanqizhen** [comment]
之前博客下的留言：

花朵旖旎
2005年12月30日 16:11 编辑
回来的时候带两个冰陀陀给我们吧。上海热得不得了-_-\’\’

回复
